packages/@cdktf/provider-generator/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -148,6 +151,9 @@ export class AttributesEmitter {
     }
 
     this.code.line(`// Getting the computed value is not yet implemented`);
+    if (type.isSet) {
+      return `cdktf.Fn.tolist(this.interpolationForAttribute('${att.terraformName}')) as any`;
```

Comment:
This really doesn't work. Perhaps we can wait for a better fix since the line below this was already broken, but doesn't feel great to add more broken things.

Reply:
Yeah 😅 I just wrapped it in `Token.asAny()` now it at least works in TypeScript (added a testcase for the edge provider test). Without the `Token.asAny()` the result of `tolist` was a list token which got stuck in the `listMapper` of the receiving resource (as that Token is an Array with a single element and so the listMapper tried to map the object). But with `Token.asAny()` the behaviour is now the same (half broken 😅) one as for lists.